### PR TITLE
york-yesno-delete

### PR DESCRIPTION
**Pull request by tpyork**
This will need some discussion on the best way to make it general.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -96,3 +96,31 @@ osf_dev_off <- function() {
...
+#' Typically embedded within a function, a question specific to the function
+#' task is prespecified for the title argument of \code{menu()}.
+#'
+#' @param question A \code{character} specific to the action needing verification.
+#' @param node_id A \code{character} indicating OSF node under consideration.
+#' @return A \code{logical} verifying intent.
+#' @examples
+#' question <- "Are you sure you want to permanently change node"
+#' node <- "z756a"
+#' yesno_menu(question, node)
```
Comment (aaronwolen) [inline R/utils.R:113]: Need to import `menu()` from the utils package.
```suggestion
#' yesno_menu(question, node)
#' @importFrom utils menu
```

File: R/utils.R [hunk 2]
```diff
@@ -96,3 +96,31 @@ osf_dev_off <- function() {
...
+#' @noRd
+
+
+yesno_menu <- function(question, node_id) {
```
Comment (aaronwolen) [inline R/utils.R]: Let's rename `node_id` to just `id` similar to other internal functions. Also, we'll probably use this w/ actions performed on non-node entities like files.

File: R/utils.R [hunk 3]
```diff
@@ -96,3 +96,31 @@ osf_dev_off <- function() {
...
+  yeses <- c("Yes", "Definitely", "For sure", "Yup", "Yeah", "I agree", "Absolutely", "Yes, 100%")
+  nos <- c("No way", "Not yet", "I forget", "No", "Nope", "Uhhhh... Maybe?", "No. That's my final answer")
+
+  qs <- c(sample(yeses, 1), sample(nos, 2))
+  rand <- sample(length(qs))
+
+  menu(qs[rand], title= sprintf("%s, %s ?", question, node)) == which(rand == 1)
```
Comment (aaronwolen) [inline R/utils.R]: Was this `node` supposed to be `node_id`?

**Review by aaronwolen — CHANGES_REQUESTED** [review]
Looks good! 👍 

I made a few suggested changes.

File: R/utils.R (hunk 3 above)
Comment (tpyork) [reply R/utils.R]: yes.  good catch !

**Review by aaronwolen — APPROVED** [review]

**Pull request merged**